website/registrations/models.py

```diff
@@ -140,9 +137,25 @@ class Meta:
         )
 
 
-class Registration(Entry):
+class Registration(Entry, Payable):
     """Describes a new registration for the association"""
 
+    @property
+    def payment_amount(self):
+        return self.contribution
+
+    @property
+    def payment_topic(self):
+        return f"Membership registration {self.membership_type} ({self.length})"
+
+    @property
+    def payment_notes(self):
+        return f"Created at {self.created_at}. Confirmed at {self.updated_at}."
```

Comment:
The notes is the member-facing information about the payment. And it should be properly human readable. This can contain the same information as is in the topic, which is only used for the treasurer imo. That's how I implemented it for everything when adding the field. Like for events: https://github.com/svthalia/concrexit/blob/7255f42587745986bec46921a7122760879606ce/website/events/models.py#L650